app/src/main/java/org/mozilla/focus/ScreenController.kt

```diff
@@ -57,11 +57,7 @@ object ScreenController {
                     .addToBackStack(null)
                     .commit()
         } else {
-            if (isSearch) {
-                SessionManager.getInstance().createSearchSession(Source.USER_ENTERED, updatedUrlStr, searchTerms)
-            } else {
-                SessionManager.getInstance().createSession(Source.USER_ENTERED, updatedUrlStr)
-            }
+            SessionManager.getInstance().createSession(Source.USER_ENTERED, updatedUrlStr)
```

Comment:
Delete the variable `searchTerms` as well, from above. The `isUrl` check above could be made into a ternary.

Reply:
We need `isUrl` to decide whether or not to send telemetry but I'll remove `searchTerms`.